src/Akeneo/Platform/Bundle/ImportExportBundle/tests/front/unit/components/JobStatusBadge.unit.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {act, screen} from '@testing-library/react';
+import {renderDOMWithProviders} from '@akeneo-pim-community/shared/tests/front/unit/utils';
+import JobExecutionStatus = require("../../../../Resources/public/js/JobStatusBadge");
+import {JobStatus} from "../../../../Resources/public/js/useJobExecutionProgress";
+
+declare global {
+  namespace NodeJS {
+    interface Global {
+      fetch: any;
+    }
+  }
+}
+
+let container: HTMLElement;
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  document.body.removeChild(container);
+  container = null;
+});
+
+
+test.each<JobStatus>(['COMPLETED', 'STOPPING', 'STOPPED', 'FAILED', 'ABANDONED', 'UNKNOWN'])
+('It displays the job progress badge without the progress', (jobStatus) => {
+
+  renderDOMWithProviders(
```

Comment:
```suggestion
  renderWithProviders(
```
Doesn't it work with the simple render?

Reply:
yes it does :)

Reply:
done works!